charts/k8tz/templates/controller.yaml

```diff
@@ -37,6 +41,29 @@ spec:
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
       containers:
+        {{- if .Values.webhook.certManager.enabled }}
```

Comment:
since the secret exist even when `webhook.certManager.enabled` is false, and the webhook anyway keep reading the certificate in any request, maybe its better to keep watcher container running always? wdyt?

Reply:
I dont think we still need the watcher container while certmanager isn't enabled, since we have already put the `checksum/config` [1] annotation to force Deployment to redeploy new pod when TLS key pair is changed.

[1] https://github.com/k8tz/k8tz/blob/master/charts/k8tz/templates/controller.yaml#L21

Reply:
but Im fine too with keeping the watcher container, just in case to keep the tls secret in-sync with http server 😄

Reply:
you are right but even with the checksum, it sound safer to run the cert-watcher. we can even remove the checksum and prevent pod recreation on upgrade when not needed. 
yet i am wondering if its worth the extra ~4.5MB memory usage. when cert-manager not in use, we generate ~14y certificate and its not something that the user should touch by hand. and pod recreation on upgrade is not that bad and upgrades are not so frequent...

maybe its better as you did in the beginning when the cert-watcher container enabled only when using cert-manager.